src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -654,11 +738,44 @@ export default {
 }
 
 .conversations-search {
-	flex-grow: 1;
-
+	transition: all 0.3s ease;
+	z-index: 1;
+	// new conversation button width : 52 px
+	// Filters button width : 44 px
+	// spacing : 1px
+	// Total : 97 px
+	width : calc(100% - 97px);
+	display : flex;
 	:deep(.input-field__input) {
 		border-radius: var(--border-radius-pill);
 	}
+	&--expanded {
+
+		// Gets expanded : 100 % - (52px + 1px)
+		width : calc(100% - 53px );
```

Comment:
Works fine in current state, but I'd like to refactor styles a bit in follow-up, because `pos:absolute` might misbehaves on different resolutions